lib/web_ui/lib/src/engine/configuration.dart

```diff
@@ -249,6 +276,7 @@ class JsFlutterConfiguration {}
 
 extension JsFlutterConfigurationExtension on JsFlutterConfiguration {
   external String? get canvasKitBaseUrl;
+  external String? get canvasKitJsFileName;
```

Comment:
This I feel is a bit too generic and unopinionated. I think instead, we should have a `canvasKitVariant` that the user specifies `'full'` or `'chromium'` or something like that. Tying this to the file name of the canvas kit JS stuff feels a little too fine grained, IMO.

Reply:
I have mixed feelings about this. But I think I agree with you. The file structure inside `canvaskit/` is an implementation detail, and we should be able to change it in the future without breaking apps.

So it's gonna be similar to web renderer i.e. 'auto' (default), 'full', 'chromium'.